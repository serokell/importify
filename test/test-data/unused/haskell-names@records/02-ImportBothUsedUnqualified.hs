module ImportBothUsedUnqualified where

import           Language.Haskell.Names (symbolName)
import qualified Language.Haskell.Names as N

main = symbolName
