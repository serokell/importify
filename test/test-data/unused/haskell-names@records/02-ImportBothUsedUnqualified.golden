module ImportBothUsedUnqualified where

import           Language.Haskell.Names (symbolName)

main = symbolName
